chia/rpc/wallet_rpc_api.py

```diff
@@ -2583,9 +2583,14 @@ async def dao_close_proposal(self, request) -> EndpointResult:
         dao_wallet = self.service.wallet_state_manager.get_wallet(id=wallet_id, required_type=DAOWallet)
         assert dao_wallet is not None
         fee = uint64(request.get("fee", 0))
+        if "genesis_id" in request:
+            genesis_id = bytes32.from_hexstr(request["genesis_id"])
+        else:
+            genesis_id = None
```

Comment:
What does `genesis_id == None` signify here?

Reply:
this is just to get the tests to run - the genesis_id code is dealt with more in the minting branch / PR